pyresample/geometry.py

```diff
@@ -1041,17 +1045,9 @@ def get_val(val, sub_val, max):
                 cols = 1
 
         # Calculate coordinates
-        target_x = np.fromfunction(lambda i, j: (j + col_start) *
-                                   self.pixel_size_x +
-                                   self.pixel_upper_left[0],
-                                   (rows,
-                                    cols), dtype=dtype)
-
-        target_y = np.fromfunction(lambda i, j:
-                                   self.pixel_upper_left[1] -
-                                   (i + row_start) * self.pixel_size_y,
-                                   (rows,
-                                    cols), dtype=dtype)
+        target_x = np.arange(col_start, col_start + cols, dtype=dtype) * self.pixel_size_x + self.pixel_upper_left[0]
+        target_y = np.arange(row_start, row_start + rows, dtype=dtype) * -self.pixel_size_y + self.pixel_upper_left[1]
+        target_x, target_y = np.meshgrid(target_x, target_y)
```

Comment:
have you checked the difference in performance with meshgrid ?

Reply:
```
In [1]: import numpy as np

In [2]: import timeit

In [4]: def test_old(rows, cols):
   ...:     dtype = np.float32
   ...:     col_start = 0
   ...:     row_start = 0
   ...:     pixel_size_x = 1000.
   ...:     pixel_size_y = 1000.
   ...:     pixel_upper_left = (-10000., 10000.)
   ...:     target_x = np.fromfunction(lambda i, j: (j + col_start) *
   ...:                                     pixel_size_x +
   ...:                                     pixel_upper_left[0],
   ...:                                     (rows,
   ...:                                      cols), dtype=dtype)
   ...:     target_y = np.fromfunction(lambda i, j:
   ...:                                     pixel_upper_left[1] -
   ...:                                     (i + row_start) * pixel_size_y,
   ...:                                     (rows,
   ...:                                      cols), dtype=dtype)
   ...:     return
   ...:                                     

In [5]: def test_new(rows, cols):
   ...:     dtype = np.float32
   ...:     col_start = 0
   ...:     row_start = 0
   ...:     pixel_size_x = 1000.
   ...:     pixel_size_y = 1000.
   ...:     pixel_upper_left = (-10000., 10000.)
   ...:     target_x = np.arange(col_start, col_start + cols, dtype=dtype) * pixel_size_x + pixel_upper_left[0]
   ...:     target_y = np.arange(row_start, row_start + rows, dtype=dtype) * -pixel_size_y + pixel_upper_left[1]
   ...:     target_x, target_y = np.meshgrid(target_x, target_y)
   ...:     return
   ...: 

In [9]: timeit.timeit('test_old(5000, 5000)', globals=globals(), number=100)
Out[9]: 35.2982305699843

In [10]: timeit.timeit('test_new(5000, 5000)', globals=globals(), number=100)
Out[10]: 6.261023567989469
```